iosApp/iosApp/Pages/Search/TextFieldObserver.swift

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
question: this looks like a new file but I think it was just moved - is there an old file that needs to be deleted?

Reply:
It was pulled out of `SearchResultView.swift`, there's still a handful of different views in there, and it used to also include this.